modules/measure/src/main/java/com/opengamma/strata/measure/rate/DefaultLookupRatesProvider.java

```diff
@@ -207,32 +210,56 @@ public FxForwardRates fxForwardRates(CurrencyPair currencyPair) {
   public IborIndexRates iborIndexRates(IborIndex index) {
     CurveId curveId = lookup.getForwardCurves().get(index);
     if (curveId == null) {
+      return historicCurve(index);
+    }
+    return IborIndexRates.of(index, getValuationDate(), marketData.getValue(curveId), timeSeries(index));
+  }
+
+  // creates a historic rates instance if index is inactive and time-series is available
+  private IborIndexRates historicCurve(IborIndex index) {
+    LocalDateDoubleTimeSeries fixings = timeSeries(index);
+    if (index.isActive() || fixings.isEmpty()) {
       throw new MarketDataNotFoundException(lookup.msgIndexNotFound(index));
```

Comment:
Not sure this is MarketDataNotFoundException in the case where index is active. it's more like MarketDataFoundException!

Reply:
This replicates the original exception. It occurs when the market data (curve) is not found for an active index.